Source/OCMock/OCMockObject.m

```diff
@@ -144,7 +144,7 @@ - (void)addExpectation:(OCMInvocationExpectation *)anExpectation
 - (void)assertInvocationsArrayIsPresent
 {
     if(invocations == nil) {
-        [NSException raise:NSInternalInconsistencyException format:@"** Cannot handle or verify invocations. This error usually occurs when a mock object is used after stopMocking has been called on it. In most cases it is not necessary to call stopMocking. If you know you have to, please make sure that the mock object is not used afterwards."];
+        [NSException raise:NSInternalInconsistencyException format:@"** Cannot handle or verify invocations on %@. This error usually occurs when a mock object is used after stopMocking has been called on it. In most cases it is not necessary to call stopMocking. If you know you have to, please make sure that the mock object is not used afterwards.", [self description]];
```

Comment:
This should probably include the memory address of the mock in question.

```suggestion
        [NSException raise:NSInternalInconsistencyException format:@"** Cannot handle or verify invocations on %@ at %p. This error usually occurs when a mock object is used after stopMocking has been called on it. In most cases it is not necessary to call stopMocking. If you know you have to, please make sure that the mock object is not used afterwards.", self, [self description]];
```

In this way the offending object can be traced back to it's point of origin with malloc stack logging.